Python/optimizer_cases.c.h

```diff
@@ -375,10 +383,11 @@
                 // replace opcode with constant propagated one and update tests!
             }
             else {
-                res = sym_new_type(ctx, &PyFloat_Type);
+                res = sym_new_type(ctx, &PyFloat_Type);stack_pointer += -2;
```

Comment:
This needs a newline. Also, nothing "escapes" here... in fact, we don't care about escaping in the optimizer at all. It's just extra wasted work here.

Reply:
We can mark anything starting with `sym_` or `optimize_` as non-escaping in the analyzer, to clean up the output a bit.
Special casing in the optimizer generator adds more complexity to the already complex code generators, so I don't think it is worthwhile.